src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -205,6 +205,32 @@
                 }
             ]
         },
+        {
+            "path": "/v1/debug/sampled_memory_profile",
+            "operations": [
+                {
+                    "method": "GET",
+                    "summary": "Get the currently sampled live memory ",
+                    "nickname": "sampled_memory_profile",
+                    "produces": [
+                        "application/json"
+                    ],
+                    "type": "array",
+                    "items": {
+                        "type": "memory_profile"
+                    },
+                    "parameters": [
+                        {
+                            "name": "shard",
```

Comment:
If you don't provide a shard, what happens?

Reply:
Output for all shards will be returned.